components/workspacekit/cmd/rings.go

```diff
@@ -340,18 +340,21 @@ var ring1Cmd = &cobra.Command{
 			}
 		}
 
-		// We deliberately do not bind mount `/etc/resolv.conf`, but instead place a copy
+		// We deliberately do not bind mount `/etc/resolv.conf` and `/etc/hosts`, but instead place a copy
 		// so that users in the workspace can modify the file.
-		err = copyResolvConf(ring2Root)
-		if err != nil {
-			log.WithError(err).Error("cannot copy resolv.conf")
-			return
+
+		copyPaths := []string{"/etc/resolv.conf", "/etc/hosts"}
+		for _, fn := range copyPaths {
+			err = copyRing2Root(ring2Root, fn)
+			if err != nil {
+				log.WithError(err).Error("cannot copy " + fn)
+				return
```

Comment:
We'll still want to try and copy the other file(s) in this case.
```suggestion
				log.WithError(err).Warn("cannot copy " + fn)
```